src/wranglerArtifactManager.ts

```diff
@@ -65,34 +90,32 @@ export async function getWranglerArtifacts(
 }
 
 /**
- * Searches for detailed wrangler output from a pages deploy
+ * Searches for a supported wrangler OutputEntry
  *
  * @param artifactDirectory
- * @returns The first pages-output-detailed found within a wrangler artifact directory
+ * @returns The first SupportedOutputEntry found within a wrangler artifact directory
  */
-export async function getDetailedPagesDeployOutput(
+export async function getOutputEntry(
 	artifactDirectory: string,
-): Promise<OutputEntryPagesDeployment | null> {
+): Promise<SupportedOutputEntry | null> {
```

Comment:
We had discussed making this function return an _array of `SupportedOutputEntry`_. This way you can simply loop over the artifacts and apply the outputs based on the artifact type

Reply:
wrangler-action should only ever have one SupportedOutputEntry, since every time the action is executed it generates a new tmpDir to be used for wranglerArtifacts https://github.com/cloudflare/wrangler-action/blob/main/src/index.ts#L25 .

Also, if it did have multiple SupportedOutputEntry then it would be possible for artifacts to override output from another one, e.g they both set `deployment-url` output